Code/Framework/AzFramework/AzFramework/Visibility/OctreeSystemComponent.cpp

```diff
@@ -418,6 +440,12 @@ namespace AzFramework
         m_root.Enumerate(frustum, callback);
     }
 
+    void OctreeScene::Enumerate(const AZ::Frustum& includeFrustum, const AZ::Frustum& excludeFrustum, const EnumerateCallback& callback) const
```

Comment:
This would be a good candidate for some unit tests.

Reply:
True. Under the hood it's really just doing frustum checks which are already tested, but would be good to add something for this function just to be sure. I'll see how complicated that will be and either add it to this PR or do a separate one.

Reply:
Ran into some complicated issues with unit tests, particularly around how the frustums created for the octree tests don't match how frustums are actually created in the culling system. I'll work on this separately so this pr can get in.

Reply:
Nevermind, got a unit test working for the octree that's as good as the rest of the octree shape tests.